src/api/app/models/user.rb

```diff
@@ -34,6 +34,12 @@ class User < ApplicationRecord
   has_many :relationships, inverse_of: :user, dependent: :destroy
   has_many :group_relationships, through: :groups, source: :relationships
 
+  has_many :direct_maintainer_relationships, -> { where(role_id: Role.hashed['maintainer'].id) }, class_name: 'Relationship'
```

Comment:
`direct` is to distinguish this relationship from the ones over the group relationship?